iroha/src/config.rs

```diff
@@ -161,14 +161,19 @@ impl Configuration {
     pub fn key_pair(&self) -> (PublicKey, PrivateKey) {
         (self.public_key, self.private_key)
     }
+
+    /// The estimated overall time it would take the blockchain to get from receiving transaction to storing it in a block on all peers.
```

Comment:
Maybe "Time estimation from receiving transaction to storage in a block on all peers."?